lib/hooks/mackerel/templates/default.html.haml

```diff
@@ -0,0 +1,5 @@
+%p
+  %span.label{class: label_class_from_build_status(payload.alert.status)}= payload.alert.status.upcase
+  #{payload.alert.monitorName} at #{payload.host.name} (#{payload.host.status}) #{payload.host.roles.first.fullname}
```

Comment:
@hibariya Thanks for reviewing!  I overlook it, and I am sorry. I updated it in `payload.host.roles.first.try(:fullname)`.